### ISSUE TITLE
Public path

### ISSUE THREAD

**Issue by tobiassjosten** [rande/swCombinePlugin#2]
The sw_include_stylesheets() (and js equivalent) doesn't seem to consider the 'public_path' setting. My CSS and JS is properly combined and placed into 'web/assets' but when the pages are rendered they link to the files like /459cb1dc4fa76b8c3948dfa4933b8890.css. As if they were in the web root.

'private_path' is set to "%SF_WEB_DIR%/assets" and 'public_path' to "/assets".

When I tried to set them to only "%SF_WEB_DIR%" and "/" then it worked. Only then it wiped everything else in my web root, like the index.php and robots.txt.

Also, possibly related, my setting for 'version' is not detected in the helper. I tried printing the $params variable in the helper but go only the the default false value for 'version'.

**Comment by tobiassjosten** [comment]
config_handlers.yml:

```
modules/*/config/view.yml:
  class: swCombineViewConfigHandler 
  param:
    private_path: %SF_WEB_DIR%/assets
    configuration:
      javascript:
        public_path:  /assets
        combine: swCombineJavascript
        driver: swDriverJSMinPlus
        filename: %s.js
        exclude:
          - /js/main.js
        packages:
          common:
            auto_include: true
            files:
              - /js/main.js
      stylesheet:
        public_path: /assets
        combine: swCombineStylesheet
        driver: swDriverCssmin
        filename: %s.css
        exclude:
          - /css/main.css
        packages:
          common:
            auto_include: true
            files:
              - /css/main.css
```

view.yml:

```
default:
  http_metas:
    content-type: text/html
  metas:
    title:  Tobias Sjösten
    robots: index, follow
  stylesheets: []
  javascripts: []
  has_layout:  true
  layout:      layout
```

app.yml:

```
swToolbox:
  swCombine:
    version: 1
```

**Comment by tobiassjosten** [comment]
The swCombineViewConfigHandler::combineValues method defined the public path in $public_path but then didn't use it until sometime later in the method. I fixed this in tobiassjosten/swCombinePlugin@de0331191a65f3173d052ce00613690013de2485.

**Comment by rande** [comment]
The bug with the private/private is now fixed. 

Can you tell me more about the version number bugs ? It seems to work fine on my side.

**Comment by tobiassjosten** [comment]
Did you accept my pull request? I can't see the commit in your repo yet.

I'll dig into the version number bug tomorrow and see what I can find out. It could very well be that I'm using the plugin incorrectly.

**Comment by tobiassjosten** [comment]
I just figured out the version number bug. What solved it was going over day six in the Jobeet tutorial. That kindly hinted that all settings in app.yml should be children of an environment parent, such as "all". I switched to this:

app.yml
    all:
      swToolbox:
        swCombine:
          version: 1

And it's working! So more of a usage problem than a plugin problem. ;)

http://www.symfony-project.org/jobeet/1_4/Doctrine/en/06#chapter_06_custom_configuration
